google/cloud/ndb/_transaction.py

```diff
@@ -23,6 +23,119 @@
 log = logging.getLogger(__name__)
 
 
+class _Propagation(object):
+    """This class aims to emulate the same behaviour as was provided by the old
+    Datastore RPC library.
+
+    https://cloud.google.com/appengine/docs/standard/python/ndb/functions#context_options
+
+    It provides limited support for transactions within transactions. It has a
+    single public method func:`handle_propagation`.
+    """
+
+    def __init__(self, propagation, join=None):
+        """Args:
```

Comment:
This docstring can be moved up to and merged with the class level docstring. Be sure to mind indentation. See other docstrings for examples, or see the [official Google guidance](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html).